pkg/security/ebpf/c/exec.h

```diff
@@ -109,6 +109,15 @@ struct bpf_map_def SEC("maps/exec_count_bb") exec_count_bb = {
     .namespace = "",
 };
 
+struct bpf_map_def SEC("maps/task_in_coredump") tasks_in_coredump = {
+    .type = BPF_MAP_TYPE_LRU_HASH,
+    .key_size = sizeof(u64),
+    .value_size = sizeof(u64),
+    .max_entries = 1024,
+    .pinning = 0,
+    .namespace = "",
+};
```

Comment:
Can we make use of `BPF_F_NO_COMMON_LRU` here ?

Reply:
applied